src/ripple/rpc/handlers/Fee1.cpp

```diff
@@ -30,7 +30,8 @@ namespace ripple
     {
         // Bail if fee escalation is not enabled.
```

Comment:
Comment no longer applicable

Reply:
Good catch. I'll check to make sure I didn't miss any others.